cohydra/network.py

```diff
@@ -91,13 +159,59 @@ def prepare(self, simulation, network_index):
         """
         logger.info('Preparing network (base IP: %s)', self.network)
 
-        if self.color is None:
-            # Color is needed for a visualization.
-            color = network_color_for(network_index, len(simulation.scenario.networks))
-            self.color = color
+        for channel_index, channel_name in enumerate(self.channels_prototypes):
+            if channel_name not in self.nodes or len(self.nodes[channel_name]) < 2:
+                logger.warning(f'{channel_name} will not be created because it has not enough nodes inside (2 at least)')
```

Comment:
Should this check rather be performed in the channel constructor itself? Does it make sense to continue operation even when we have hit this case?